introduction.md

```diff
@@ -6,21 +6,24 @@ secure defaults and a great developer experience.
 
 It's built on V8, Rust, and Tokio.
 
+It can be considered a modern alternative to Node.js. s
```

Comment:
Typo at the end of the line.  Also, I am not totally sure about the positioning with this statement, @alonbonder thoughts?

Reply:
I just removed this.